### PR TITLE
Make certs for Gitpod installation on harvester available 

### PR DESCRIPTION
**Pull request by meysholdt**
## Description
This fixes two issues:
* the issuer and dns-sa for Certmanger were not installed when the VM existed already but did not have them yet. 
* the gitpod installation did not yet get a certificate. 

## Related Issue(s)

Fixes # NONE

## How to test
1. run ` werft run github -a with-vm=true`
2. get a kubectx for k3s
3. confirm that all pods the gitpod installation came up. they only do that if all needed certs are present.

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **corneliusludmann** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @corneliusludmann` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: .werft/build.ts
Comment (mads-hartmann) [inline .werft/build.ts:942]: I would prefer naming the arguments `previewNamespace` and `certsNamespace` or something to that affect so it is clear that both args are namespace names. I also generally prefer option arguments so we get named parameters, e.g. 

```ts
export async function issueMetaCerts(options: {previewNamespace: string, certsNamespace: string, domain: string})
```

**Review by mads-hartmann — APPROVED** [review]
Awesome, thanks! I added the hold label and added a tiny suggestion, but I don't feel strongly enough about it to request changes, so feel free to remove the label if you don't agree with the proposed changes ☺️

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 0fe0f9551e8b9e295735b6549419e5b01d9f171a</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7480#pullrequestreview-846394803" title="Approved">mads-hartmann</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **corneliusludmann** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @corneliusludmann` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
New changes are detected. LGTM label has been removed.

File: .werft/build.ts
Comment (meysholdt) [reply .werft/build.ts]: Good suggestion! applied :)

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7480#pullrequestreview-846394803" title="Approved">mads-hartmann</a>*

*No associated issue*. Requirement bypassed by manually added approval.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by mads-hartmann — APPROVED** [review]
Changes still look good ☺️

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: f5e974e9d0ce0af22e407400fa31e3fb1fdc84be</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7480#pullrequestreview-846432296" title="Approved">mads-hartmann</a>*

*No associated issue*. Requirement bypassed by manually added approval.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
